hooks/_common.sh

```diff
@@ -55,6 +55,44 @@ function common::parse_cmdline {
   done
 }
 
+#######################################################################
+# Expand environment variables definition into their values in '--args'.
+# Support expansion only for ${ENV_VAR} vars, not $ENV_VAR.
+# Globals (modify):
+#   ARGS (array) arguments that configure wrapped tool behavior
+#######################################################################
+function common::parse_and_export_env_vars {
+  local -r len=${#ARGS[@]}
+  local arg
+
+  for ((i = 0; i < len; i++)); do
+    arg="${ARGS[$i]}"
```

Comment:
Tempting fate once more:
```suggestion
  for arg_idx in "${!ARGS[@]}"; do
    arg="${ARGS[$arg_idx]}"
```